### PR TITLE
Ability to build and store GCE machine images

### PR DESCRIPTION
**Pull request by jrroman**
Changes:
- Updating build-image.sh to allow for gce builds and pushes to gce storage. 
- Script to convert the qcow2 image into raw format, compress it, store in gcp, and create a machine image. 
- Added `scripts/elotl/packages-gce` for host packages
- Updated the repositories file to use alpine 3.11 repos

### CODE REVIEW

File: scripts/build-image.sh [hunk 1]
```diff
@@ -122,6 +124,16 @@ if [[ "$CLOUD_PROVIDER" == "azure" ]] && [[ "$NO_IMAGE" = false ]]; then
     echo "OK"
 fi
 
+if [[ "$CLOUD_PROVIDER" == "gce" ]] && [[ "$NO_IMAGE" = false ]]; then
+    REQUIRED_PROGRAMS="$REQUIRED_PROGRAMS gsutil gcloud jq"
```
Comment (justnoise) [inline scripts/build-image.sh:128]: Do we overwrite this below?  Probably need to update the `REQUIRED_PROGRAMS="qemu-img qemu-nbd"` to be like you have it here, `REQUIRED_PROGRAMS="$REQUIRED_PROGRAMS qemu-img qemu-nbd"`

File: scripts/elotl/repositories [hunk 2]
```diff
@@ -1,2 +1,2 @@
-https://nl.alpinelinux.org/alpine/v3.9/main
-https://nl.alpinelinux.org/alpine/v3.9/community
+https://nl.alpinelinux.org/alpine/v3.11/main
```
Comment (justnoise) [inline scripts/elotl/repositories:1]: :+1:

File: scripts/build-image.sh (hunk 1 above)
Comment (jrroman) [reply scripts/build-image.sh:128]: Oh that is a very good point, let me make that update

Comment (jrroman) [reply scripts/build-image.sh:128]: fixed in my recent commit.

File: scripts/elotl/repositories (hunk 2 above)
Comment (ldx) [inline scripts/elotl/repositories:1]: If you want to move to 3.11, we should also update the commit we check out from alpine-make-vm-image: https://github.com/elotl/itzo/blob/master/scripts/build-image.sh#L157

Comment (jrroman) [reply scripts/elotl/repositories:1]: Good catch I have addressed that in the most recent commit. I added a checkout for the commit at HEAD of master from alpine-make-vm-image repo so we can stay stable.

**Pull request merged**